Docker-compose-web-srv/debian-apache2/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.10
+
+RUN apt-get update -y; \
+    apt-get install -y apt-utils; \
```

Comment:
use one apt-get install instead of three:
apt-get install -y apt-utils apache2 apache2-utils

Reply:
I completely agree. At some moment I kept it in my mind, but forgot at the last stage - when everything worked.